content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,546 @@
+curl -LO https://raw.githubusercontent.com/containerd/containerd/main/containerd.service
+```
+
+Configure:
+
+```shell
+sudo mkdir -p /etc/containerd
+
+sudo tee /etc/containerd/config.toml <<EOF
+version = 2
+
+[plugins."io.containerd.grpc.v1.cri".cni]
+
+    # binaries
+    bin_dir = "/opt/cni/bin"
+
+    # configure pod networking
+    conf_template = "/etc/cni/cni.template"
+EOF
+```
+
+Install:
+
+{{< tabs name="install_containerd" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+sudo tar Cxzvf /usr/local/ containerd-1.7.22-linux-amd64.tar.gz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+sudo tar Cxzvf /usr/local/ containerd-1.7.22-linux-arm64.tar.gz
+{{< /tab >}}
+{{< /tabs >}}
+
+```shell
+sudo cp containerd.service /etc/systemd/system/
+sudo systemctl daemon-reload
+sudo systemctl enable --now containerd.service
+```
+
+Quick test:
+
+```shell
+systemctl is-active containerd.service
+```
+
+The output is similar to:
+
+```
+active
+```
+
+Detailed service check:
+
+```shell
+sudo journalctl -f -u containerd.service
+```
+
+**runc**
+
+Visit [latest release](https://github.com/opencontainers/runc/releases/latest)
+(external link).
+
+Download:
+
+{{< tabs name="download_runc" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO https://github.com/opencontainers/runc/releases/download/v1.1.14/runc.amd64
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/opencontainers/runc/releases/download/v1.1.14/runc.arm64
+{{< /tab >}}
+{{< /tabs >}}
+
+Install:
+
+{{< tabs name="install_runc" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+chmod +x runc.amd64
+sudo cp runc.amd64 /usr/sbin/runc
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+chmod +x runc.arm64
+sudo cp runc.arm64 /usr/sbin/runc
+{{< /tab >}}
+{{< /tabs >}}
+
+Quick test:
+
+```shell
+runc --version
+```
+
+The output is similar to:
+
+```
+runc version 1.1.14
+...
+```
+
+### Kubelet
+
+Download the [latest stable release](/releases/download/) of the Kubelet.
+
+{{< tabs name="download_kubelet" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubelet"
+{{< /tab >}}
+{{< /tabs >}}
+
+Configure:
+
+{{< note >}}
+This tutorial uses the `10.241.1.0/24` network range (which is a private network).
+Make sure you don't have any conflicts with an existing network. If necessary,
+use a network that is available to you.
+{{< /note >}}
+
+```shell
+sudo mkdir -p /etc/kubernetes/manifests
+```
+
+```shell
+sudo tee /etc/kubernetes/kubelet.yaml <<EOF
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+authentication:
+  webhook:
+    enabled: false
+authorization:
+  mode: AlwaysAllow
+enableServer: false
+logging:
+  format: text
+readOnlyPort: 10250
```

Comment:
Add a note somewhere that this is an insecure configuration. Is there a page you can link to on setting up Kubelet authentication?

Reply:
I will add a note. AFAIK, we only have the [reference docs](https://deploy-preview-48094--kubernetes-io-main-staging.netlify.app/docs/reference/command-line-tools-reference/kubelet/) with _all_ the options listed in one place.

Reply:
Use a [caution](https://kubernetes.io/docs/contribute/style/style-guide/#caution) callout for this.

Reply:
Done